### PR TITLE
переработка кода

### PR DESCRIPTION
**Pull request by qwertyppp11**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/qwertyppp11/online-strategic-game/pull/32" target="_blank">ReviewNinja

File: api/core/System.php [hunk 1]
```diff
@@ -118,10 +117,10 @@ protected function register ( $options ) {
             $uname = htmlspecialchars($uname);
 
             //пользователь
-            $this->db->custom_no_return( "INSERT INTO `users` (`id`, `uname`, `upass`, `umail`, `reg_time`) VALUES (NULL, ?, ?, ?, ?)",
+            $this->database->custom_no_return( "INSERT INTO `users` (`id`, `uname`, `upass`, `umail`, `reg_time`) VALUES (NULL, ?, ?, ?, ?)",
                 array( $uname, password_hash($options['upass'], PASSWORD_DEFAULT), $options['umail'], time() )
             );
-            $id = $this->db->custom(
+            $id = $this->database->custom(
```
Comment (qwertyppp11) [inline api/core/System.php]: Codacy Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://www.codacy.com/app/qwertyppp11/online-strategic-game/file/2702880806/issues/source?bid=3342332&fileBranchId=3342352#l123)

File: web-client/js/game.js [hunk 2]
```diff
@@ -7,10 +7,11 @@ tabs.children('div:not(:first)').hide();
 tabs.children('div:first').css( 'display', 'flex' );
 tabs.children('#tab-control').children('li:first').addClass( 'tab-active' );
 tabs.children('#tab-control').children('li').click(function ( event ) {
+    var target;
     if ( event.target.tagName == 'SPAN' ) {
-        var target = $( event.target ).parent();
+        target = $( event.target ).parent();
```
Comment (qwertyppp11) [inline web-client/js/game.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/qwertyppp11/online-strategic-game/file/2703141884/issues/source?bid=3342332&fileBranchId=3342352#l12)

File: web-client/js/game.js [hunk 3]
```diff
@@ -7,10 +7,11 @@ tabs.children('div:not(:first)').hide();
 tabs.children('div:first').css( 'display', 'flex' );
 tabs.children('#tab-control').children('li:first').addClass( 'tab-active' );
 tabs.children('#tab-control').children('li').click(function ( event ) {
+    var target;
     if ( event.target.tagName == 'SPAN' ) {
-        var target = $( event.target ).parent();
+        target = $( event.target ).parent();
     } else {
-        var target = $( event.target );
+        target = $( event.target );
```
Comment (qwertyppp11) [inline web-client/js/game.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/qwertyppp11/online-strategic-game/file/2703141884/issues/source?bid=3342332&fileBranchId=3342352#l14)

File: api/core/System.php (hunk 1 above)
Comment (qwertyppp11) [inline api/core/System.php]: Codacy Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://www.codacy.com/app/qwertyppp11/online-strategic-game/file/2703141902/issues/source?bid=3342332&fileBranchId=3342352#l123)

**Pull request merged**